src/table/virtualized-table/Body.tsx

```diff
@@ -135,7 +135,7 @@ const Body = forwardRef<BodyRef, BodyProps>((props, ref) => {
       data-key="body"
```

Comment:
Would guess that it was to "tag" the element so that it could be identified in the browser when inspecting. But it does not work. `VariableSizeGrid` just ignores it.